### PR TITLE
Update CoC

### PR DESCRIPTION
**Pull request by orendon**
image

### CODE REVIEW

File: source/code_of_conduct.html.haml [hunk 1]
```diff
@@ -66,49 +52,38 @@ title: RubyConf - Colombia
...
-          %p This code of conduct and its related procedures also applies to unacceptable behavior occuring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+          %p We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues—online and in- person—as well as in all one-on-one communications pertaining to community business.
 
         .accordion__header
-          %h2.accordion__header--title 9. Contact info
+          %h2.accordion__header--title 9. Contact Info
         .accordion__body
-          %p [YOUR CONTACT INFO HERE -- this should be a single person or a small team who can respond to issues directly]
+          %p
+            %a{:href => "mailto:rubyconfco@gmail.com"} Governing Body
```
Comment (sogamoso) [inline source/code_of_conduct.html.haml]: Do we already have this email account?

Comment (orendon) [inline source/code_of_conduct.html.haml]: yep we own it. We could create `organizers@rubyconf.co` but I don't have domain/email access, wdyt?

Comment (robertomiranda) [inline source/code_of_conduct.html.haml]: @orendon done :+1:

Comment (sogamoso) [inline source/code_of_conduct.html.haml]: Thanks @robertomiranda, @orendon mind updating the email?

Comment (orendon) [inline source/code_of_conduct.html.haml]: Done https://github.com/RubyConfCo/rubyconf.co/commit/8e2c27bab7452ad947cf03ceb39f24a990e48e27

**Pull request merged**